src/app/media-wall/media-wall-header/media-wall-header.component.html

```diff
@@ -1,10 +1,19 @@
 <div class="header-wrapper">
 	<div class="header">
-		<a routerLink="/">
 			<div class="logo">
-				loklak
+				<a routerLink="/">
+					loklak
+				</a>
 			</div>
-		</a>
-		<div class="title">{{query}}</div>
+		<div class="title">
+			<span class="query">{{query}}
+			</span>
+			</div>
+		<div class="menu" [class.mouse-moving]="showHideMenu">
+			<div class="option" toggleFullscreen>
```

Comment:
Wrap this inside a button element, as currently this button is not accessible
Always keep `a11y` in mind while writing components, also please make sure you provide proper role and name to the button.